src/gpodder/model.py

```diff
@@ -293,7 +320,7 @@ def from_podcastparser_entry(cls, entry, channel):
         episode.link = entry['link']
         episode.episode_art_url = entry.get('episode_art_url')
         if entry.get('description_html'):
-            episode.description = ''
+            episode.description = util.remove_html_tags(entry['description_html'])
```

Comment:
I dont remember even making this change

Reply:
OH! I made this change since I thought it was kind of weird to just set the description to nothing, because it would mean extra logic would be needed to determine which one to use later on. im not sure if this is breaking though so I can remove it if needed

Reply:
Only one of those fields should be set, setting both bloats the database and causes gpodder to take longer to startup. cache_text_description() is called for every episode at startup to initialize episode._text_description which is used by anything wanting a text-only description. And html_description() is used by anything wanting an html description.

Also, I had a bad `chown -R root:root /` accident as `root` today and will be busy repairing my system for a while. So it might be 2023 before I can review this.

Reply:
What I remembered then. Please @Happyrobot33 rollback this change.

Reply:
resolved in [e9b0dce](https://github.com/gpodder/gpodder/pull/1432/commits/e9b0dce9e92a74534cdf860e6396b90bcfddba94)